.github/workflows/prod-deploy.yml

```diff
@@ -0,0 +1,51 @@
+name: Beta Deployment Workflow
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  beta-deployment:
+    if: >-
+      github.event.issue.pull_request &&
+      contains(github.event.comment.body, 'Approve Prod Deployment')
```

Comment:
Can anyone write this line and approve prod?

Reply:
Yes Peter, we can control who can approve just by adding their gh alias in the above condition. For example github.actor == 'Divyaasm'.